app/views/pages/personal_dashboard.html.erb

```diff
@@ -30,31 +26,35 @@
       </li>
       <% end %>
     </ul>
+    <%= link_to '', new_user_plan_path %>
```

Comment:
why do we have a link to here with no destination and a button in the next line?